File: nbui-drag/src/main/java/cn/rubintry/nbui/drag/PositionHandler.java
package cn.rubintry.nbui.drag;

import android.graphics.Point;
import android.graphics.RectF;

public class PositionHandler {

    public RectF getRectF(NBFloatView view) {
        if(view.position == null){
            return new RectF();
        }
        Point position = view.position;
        return new RectF((float) position.x, (float) position.y , (float) position.x + view.mWidth , (float) (position.y + view.mHeight));
    }
}

File: nbui-drag/src/main/java/cn/rubintry/nbui/drag/PositionHandler.kt
package cn.rubintry.nbui.drag

import android.graphics.RectF


internal class PositionHandler {

    fun getRectF(view: NBFloatView): RectF {
        val position = view.position ?: return RectF()
        return RectF(position.x.toFloat() , position.y.toFloat() , position.x.toFloat() + view.mWidth , position.y.toFloat() + view.mHeight)
    }
}